src/js/components/Carousel/doc.js

```diff
@@ -25,6 +25,9 @@ export const doc = Carousel => {
       will loop through all children indefinitely.`),
     initialChild: PropTypes.number.description(`If specified, the index of
       the first element to be shown. Defaults to 0.`),
+    controls: PropTypes.oneOf([PropTypes.bool, 'arrows', 'selectors'])
```

Comment:
```suggestion
    controls: PropTypes.oneOf([PropTypes.bool, PropTypes.oneOf(['arrows', 'selectors'])])
```
I'm thinking this will generate a better Readme file and be more proptype accurate.

Reply:
true true, my bad